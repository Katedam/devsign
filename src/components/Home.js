import React from 'react';
import UserDetails from './UserDetails';
import PopularChirps from '../containers/PopularChirps';
import Nav from './Nav';
import { Section, Main } from '../styles/mainStyle';

const  user = {
  name: 'Kaiya',
  handle: '@ladybeard',
  profileImage: 'http://capepremierrealty.com/wp-content/uploads/2018/11/generic-profile-icon-7.jpg.png',
  bannerImage: 'https://upload.wikimedia.org/wikipedia/commons/b/bc/Saddle_tor_to_Hey_tor_pano.jpg',
  bio: 'Fullstack MERN Developer / Animal Lover / Hiker',
  location: 'Portland, OR'
};

function Home() {
  const { name,  handle, profileImage, bannerImage, bio, location } = user;
  return (
    <>
    <Nav />
    <Main>
      <UserDetails
        name={name}
        handle={handle}
        profileImage={profileImage}
        bannerImage={bannerImage}
        bio={bio}
        location={location}
      />
      <Section>
        <PopularChirps />
      </Section>
    </Main>
    </>
  );
}

export default Home;
