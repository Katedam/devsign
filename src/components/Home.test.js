import React from 'react';
import Home from './Home';
import { shallow } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';

describe('Home component', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(
      <MemoryRouter>
        <Home />
      </MemoryRouter>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
