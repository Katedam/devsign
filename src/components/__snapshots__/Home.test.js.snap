// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component matches a snapshot 1`] = `
Array [
  <header
    className="sc-bwzfXH ikzJqp"
  >
    <h1>
      Chirp
    </h1>
  </header>,
  <nav
    className="sc-dnqmqq sc-iwsKbI gUAlru"
  >
    <ul
      className="sc-gZMcBi cwXRZz"
    >
      <a
        className="sc-bdVaJa cVFclJ"
        href="/home"
        onClick={[Function]}
      >
        <li
          className="sc-gqjmRU fMjxHK"
        >
          Home
        </li>
      </a>
      <a
        className="sc-bdVaJa cVFclJ"
        href="/profile"
        onClick={[Function]}
      >
        <li
          className="sc-gqjmRU fMjxHK"
        >
          Profile
        </li>
      </a>
      <a
        className="sc-bdVaJa cVFclJ"
        href="/followers"
        onClick={[Function]}
      >
        <li
          className="sc-gqjmRU fMjxHK"
        >
          Followers
        </li>
      </a>
    </ul>
    <form
      className="sc-bxivhb sc-EHOje hZYZKW"
    >
      <label
        className="sc-ifAKCX hrWLvt"
        htmlFor="search"
      >
        Search
      </label>
      <input
        className="sc-bZQynM dqSVUO"
        id="search"
        readOnly={true}
        type="text"
        value=""
      />
      <label
        className="sc-ifAKCX hrWLvt"
        htmlFor="searchButton"
      />
      <button
        className="sc-gzVnrw ccvAWg"
        id="searchButton"
      >
        Search Chirp
      </button>
    </form>
  </nav>,
  <main
    className="sc-htpNat jrSCQS"
  >
    <aside
      className="sc-VigVT bGQLkE"
    >
      <img
        className="sc-fjdhpX dbHkHM"
        name="bannerImage"
        src="https://upload.wikimedia.org/wikipedia/commons/b/bc/Saddle_tor_to_Hey_tor_pano.jpg"
      />
      <img
        className="sc-jTzLTM fkBueS"
        name="profileImage"
        src="http://capepremierrealty.com/wp-content/uploads/2018/11/generic-profile-icon-7.jpg.png"
      />
      <h2>
        Kaiya
         
        <span
          className="sc-cSHVUG crXluK"
        >
          @ladybeard
        </span>
      </h2>
      <p
        className="sc-kAzzGY sc-chPdSV dLxFcj"
      >
        Fullstack MERN Developer / Animal Lover / Hiker
      </p>
      <p
        className="sc-kAzzGY sc-chPdSV dLxFcj"
      >
        Portland, OR
      </p>
    </aside>
    <section
      className="sc-kGXeez jFjSmc"
    >
      <section>
        <ul
          className="sc-gZMcBi cwXRZz"
        >
          <li>
            <article
              className="sc-kpOJdX dvXSoz"
            >
              <header>
                <img
                  className="sc-jTzLTM fkBueS"
                  src="http://cdn.onlinewebfonts.com/svg/img_191958.png"
                />
                <h2>
                  Ray
                   
                  <span
                    className="sc-cSHVUG crXluK"
                  >
                    @stingraymond
                  </span>
                </h2>
              </header>
              <p
                className="sc-kgoBCf HaKVQ"
              >
                tweet tweet tweet
              </p>
            </article>
          </li>
          <li>
            <article
              className="sc-kpOJdX dvXSoz"
            >
              <header>
                <img
                  className="sc-jTzLTM fkBueS"
                  src="http://cdn.onlinewebfonts.com/svg/img_191958.png"
                />
                <h2>
                  Kaiya
                   
                  <span
                    className="sc-cSHVUG crXluK"
                  >
                    @ladybeard
                  </span>
                </h2>
              </header>
              <p
                className="sc-kgoBCf HaKVQ"
              >
                tweet tweet tweet
              </p>
            </article>
          </li>
          <li>
            <article
              className="sc-kpOJdX dvXSoz"
            >
              <header>
                <img
                  className="sc-jTzLTM fkBueS"
                  src="http://cdn.onlinewebfonts.com/svg/img_191958.png"
                />
                <h2>
                  Randy
                   
                  <span
                    className="sc-cSHVUG crXluK"
                  >
                    @randyroute
                  </span>
                </h2>
              </header>
              <p
                className="sc-kgoBCf HaKVQ"
              >
                tweet tweet tweet
              </p>
            </article>
          </li>
        </ul>
      </section>
    </section>
  </main>,
]
`;
